app/javascript/src/js/mdPreviewShare.js

```diff
@@ -0,0 +1,15 @@
+function generateLink(content) {
```

Comment:
Another minor nitpick 😄  can we rename this file from from `mdPreviewShare.js` to `previewShare.js` please? we could potentially reuse these methods for any sort of content.

Reply:
You got it!